src/NadekoBot/data/strings/commands/commands.en-US.yml

```diff
@@ -2040,4 +2040,8 @@ economy:
 purgeuser:
   desc: "Purge user from the database completely. This includes currency, xp, clubs that user owns, waifu info"
   args:
-    - "@Oblivion"
\ No newline at end of file
+    - "@Oblivion"
+quotesearchkey:
+  desc: "Searches for quote using keyword 15 Per page."
+  args:
+    - "test"
```

Comment:
Can you give a better name to this arg, like "keyword" or "query"

Reply:
changed to keyword